app/src/main/java/cz/quanti/razym/rocketapp/presentation/RocketListViewModel.kt

```diff
@@ -3,11 +3,25 @@ package cz.quanti.razym.rocketapp.presentation
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
 import cz.quanti.razym.rocketapp.domain.RocketModel
 import cz.quanti.razym.rocketapp.domain.RocketsRepository
+import kotlinx.coroutines.launch
 
 class RocketListViewModel(
-    repo: RocketsRepository
+    private val repo: RocketsRepository
 )  : ViewModel() {
-    val rocketLiveData: LiveData<List<RocketModel>> = MutableLiveData(repo.getRockets())
+    private val _rocketLiveData: MutableLiveData<Resource<List<RocketModel>>> = MutableLiveData()
+    val rocketLiveData: LiveData<Resource<List<RocketModel>>> = _rocketLiveData
+
+    init {
+        refreshRockets()
+    }
+
+    fun refreshRockets() {
```

Comment:
refresh v nazvu metody mi evokuje to, ze uz nejaka data mam, coz neni pravda v pripade kdy se to vola z init bloku. za me by byl lepsi naming proste loadRockets